test/bridgeAdmin.ts

```diff
@@ -0,0 +1,226 @@
+import { expect } from "chai";
+import { ethers, upgrades } from "hardhat";
+import { SignerWithAddress } from "hardhat-deploy-ethers/signers";
+
+import { AddressOne, convertToUnit } from "../helpers/utils";
+import {
+  AccessControlManager,
+  LZEndpointMock,
+  LZEndpointMock__factory,
+  XVS,
+  XVSBridgeAdmin,
+  XVSProxyOFTDest,
+  XVSProxyOFTDest__factory,
+  XVS__factory,
+} from "../typechain";
+
+describe("Bridge Admin: ", function () {
+  const localChainId = 1;
+  const remoteChainId = 2;
+  const singleTransactionLimit = convertToUnit(10, 18);
+  const maxDailyTransactionLimit = convertToUnit(100, 18);
+
+  let LZEndpointMock: LZEndpointMock__factory,
+    ProxyOFTV2Dest: XVSProxyOFTDest__factory,
+    RemoteTokenFactory: XVS__factory,
+    remoteEndpoint: LZEndpointMock,
+    remoteOFT: XVSProxyOFTDest,
+    bridgeAdmin: XVSBridgeAdmin,
+    remotePath: string,
+    alice: SignerWithAddress,
+    deployer: SignerWithAddress,
+    accessControlManager: AccessControlManager,
+    remoteToken: XVS;
+
+  async function grantPermissions() {
```

Comment:
Could this be returned as a fixture? That way you can retrieve the contracts with this state for testing

Reply:
[15bd3cb](https://github.com/VenusProtocol/token-bridge/pull/1/commits/15bd3cbac794fb891efbde9655e85692458181a5)